app/assets/javascripts/experiments/allowed_infrastructures.js.coffee

```diff
@@ -0,0 +1,246 @@
+class window.AllowedInfrastructures
```

Comment:
Ogólny komentarz do tego pliku to: "jakby kiedyś był czas" to trzeba by zrobić wspólną klasę bazową dla tych widgetów drzew (pisałem kiedyś o tym).